src/pages/Admin/Venue/VenueDetailsForm.tsx

```diff
@@ -0,0 +1,351 @@
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router-dom";
+import Bugsnag from "@bugsnag/js";
+import * as Yup from "yup";
+
+import { IS_BURN } from "secrets";
```

Comment:
Do we actually need `IS_BURN` check at all?

Reply:
I have removed all `IS_BURN` uses in other PR. This will be cleaned up by merge from staging.